### ISSUE TITLE
Which document formats are accepted for GC web accessibility?

### ISSUE THREAD

**Issue by marieeved** [wet-boew/wet-boew#7966]
Hi,
I'm reading the [Standard on Web Accessibility](http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=23601) and the [Web Content Accessibility Guidelines (WCAG) 2.0](http://www.w3.org/TR/WCAG20/) and I didn't find any reference to which document formats (Word, PDF, PWP...) are accepted or forbidden to respect this standard.
or must we always publish everything in HTML?

Can you help me?
Thank you.

**Comment by rlambert27** [comment]
You should have a read on "accessibility supported" :

https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-accessibility-support-head

You can publish in any format that meets those requirements. Typically for the web on a GoC website that would mean HTML and PDF. You could publish in other formats like Flash but typically those have been avoided in the past.

**Comment by marieeved** [comment]
thanks for the information.

Do you know if there is an easy way to evaluate if an already existing PDF on my website is accessible or not?
because there is so much rules in the WCAG and it's hard to know all the examples that are not "accessible"...

**Comment by rlambert27** [comment]
An easy way? Well Adobe Pro does have a built-in accessibility checker:

https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html

It requires you have the pro version of Adobe though. However a full conformance verification would require someone to manually evaluate your PDF against the WCAG 2.0 sufficient techniques for PDFs.

At ESDC the WS CoE has experts who can perform full conformance evaluations. If you'd like us to have a look at one don't hesitate to send me an email.

**Comment by marieeved** [comment]
that's what I just founded... yes, we have pro version. I just made a test on one of my PDF. it's really fast but after it depends on how many errors do you have...

well, the problem is that we have **a lot** of PDF on our intranet and we realized that there are probably all not accessible...

I learn about all this to write guidelines to our authors of web contents to start again on the right track (for future contents).

What does mean "WS CoE"? 
thanks.

**Comment by shawnthompson** [comment]
@marieeved How are you creating your PDFs. If you're exporting them from MS Word, there's an option you have to select to have your document tagged properly.

[Create accessible PDFs](https://support.office.com/en-us/article/Create-accessible-PDFs-064625e0-56ea-4e16-ad71-3aa33bb4b7ed)

*Web Standards Centre of Expertise (WS CoE)

**Comment by marieeved** [comment]
Yes, I think that the PDFs is created from a Word document (it's not me that created these PDF, it's our contents authors, ex: someone from a business lines). Thanks for the link. 

other questions: 
1a- HTML and PDF are common formats for GoC websites. Does it means that for a content, we can have only PDF format? (by example for a guide of several pages) or we must to have the content absolutly in HTML and if we like we can also have this content in a PDF?

2a-You said we can publish in any formats if the document is accessible. but, by example, if I have a PowerPoint/Excel, is it better to still transform it in PDF?

2b-if I have a map or a diagram, is it better to have it in an image or in a PDF?

3-by your experience, what is the easiest/fastest? 
make a document (pdf, word...) accessible or create this content in HTML?

thank you very much.

**Comment by jvanulde** [comment]
The default format should be HTML. PDF's can be generated on demand. There is no need to package documents up into vendor specific formats unless users request it.

**Comment by rlambert27** [comment]
1a - Well technically you can it as a PDF only if it meets conformance but see what @jvanulde said, you should probably use HTML only.

2a - It depends where you are publishing. If you are publishing external (to the internet) then HTML / PDF only, if you are publishing internal (to the intranet) then it's the wild west depending on your department.

2b - I guess it depends but typically an image with a long description.

3 - Create the content in HTML.

**Comment by marieeved** [comment]
thanks a lot for the information. It's very helpful.

It's for an intranet.

pt1 and 3. it terrifies me... almost all our content is links towards documents (PDF).

Yes, sometimes, we don't have choice to have documents in PDF... because our intranet it's not just to give information. We want that our intranet is a work tool (employees must have access to all info/docs to do a task or a demand).
Example, on [this page](http://edsc.prv/en/service-canada/quebec/us/people/awards-excellence/index.shtml)
we have information to learn how to propose a nominee. 
Employees have to complete forms and send them to someone in charge.
I understand that the PDF for the "tips" could be transformed in HTML. it's just infofrmation. It would just need time to create.
but for me the forms must to stay in PDF. They can't be in HTML?
I have other cases where people have to send letters. They can't be in HTML either.

It makes me think : if the PDF is on a common documentary share, the accessibility web standards must to be respected too or not?

**Comment by Bgolden67** [comment]
The basic rule of thumb we have in our Department is HTML first and foremost for the public facing Internet site. If the clients requests a PDF linked from the HTML we question it's purpose. A PDF document that is nothing more than the WORD content saved to PDF is what we try to avoid. Most often the client says they want a PDF for printing but since the web page print function works well to remove template styles there is no real need for that. If the PDF was a booklet created for print and the content is being adapted for the web, then we will link to the PDF version as it's a true print intended design.

We have implemented fully accessible PDF Forms that meet the guidelines and have been tested to be accessible.

For our Intranet, it used to be a wild west scenario as @rlambert27 described, but over the last 2 years a total redesign took place and the intranet team is starting to enforce more accessibility than ever following similar requirements as our internet site. The reson for this is simple, the department has a Duty to accommodate, which very much like the accessibility requirements for external sites, applies internally to the employees who may also be using adaptive technologies and these inaccessible PDFS will prevent them from completing a task.

**Comment by rlambert27** [comment]
I would contact your own WS CoE:

http://publiservice.tbs-sct.gc.ca/ws-nw/wc-cw/wsce-cenw-eng.asp

Not sure which department you are from but it should be listed. Each department has their own policy on internal documents and how accessible they should be. I know of several departments that have literally no requirements on documents posted on the intranet.

Your WS CoE rep. should be more aware of the rules within your department.

**Comment by Bgolden67** [comment]
@rlambert27 I think many of the WS CoEs are being forgotten by the departments since the change in standards over the past couple years, unless they are completely involved the way I am here at INAC

**Comment by marieeved** [comment]
Hi @rlambert27,
it's you the team leader of ESDC... :)

probably last example/question:

Often, we have documents (like PDF) that I think they can't be transformed in HTML...
In this [page](http://edsc.prv/fr/service-canada/quebec/ressources/securite/test-norme-filtrage.shtml), I think it will cover most common types of documents that we have on our intranet. 
***tell me if you can't see it.**

I will show you several documents (in order of appearance on the page), please tell me if I'm wrong about accessible solutions for them : 
1-"Guide de référence pour les gestionnaires" : must be transformed in HTML because it's just  informations/tips?
2-"Cheminement des demandes de cotes de sécurité" : like it's a diagram, it could be transformed in an image with a long description tag or in a normal text in HTML (step 1, 2, ...)?
3-"Formulaire de vérification de sécurité, de consentement et d’autorisation du personnel (TBS 330-23)": it belongs to the National, so it's not our responsability, right?
4-"Formulaire de prise de références" : it's a document to fill out and print. I don't know what to do with this kind of document. Have you a solution?
5-"Pour le gestionnaire" : I think it could be transformed in image, but the long description would be very long because it represents a help tool to fill out a form. If a person can't see it, it will be hard to explain just in words... Have you a better solution? or in that particular case, it's better to offer a person contact that will help directly the person to fill out the form?
6-"Lettre d'information à remettre au candidat": a letter to print to give to a future employee (who doesn't have access to intranet...). I don't know what to do with this kind of document either. Have you a solution?
7-"Présentation PowerPoint pour la séance d’information sur la sécurité": We give the two options on the page: a link towards the pwp like this and a link towards only the text in an HTML page? Have you a better solution?

Thank you in advance for your time and your help. It's really appreciated.

**Comment by rlambert27** [comment]
@Bgolden67 We're certainly still involved also. We're tightly integrated into the software release process and our shop is a mandatory stop in the process.

@marieeved 

1 - HTML, no reason to use a PDF.
2 - HTML, can easily be represented in text.
3 - PDF is fine as long as it passes conformance. WCAG 2.0 doesn't make the distinction to whom it belongs, if it's on the web it should be made accessible.
4 - HTML or PDF but I would stick with HTML. You can fill it out on the HTML page and simply print after.
5 - Yes you could offer a contact person but you could also use HTML. Having text below the image indicating help with the form certainly wouldn't be a burden.
6 - HTML but if you need to send it out then yes you could use a PDF. Just make sure it's accessible :-)
7 - I only noticed one link not two, the one for PPT. That can easily be done in HTML as well.

Hope that helps! If you need further help for our intranet stuff you can send me an email directly if you are at ESDC :-)

**Comment by marieeved** [comment]
Thanks to all for your precious help.

@rlambert27 : thank you for the offer. If I have further questions, I will contact you by email.
